tools/props/Versions.props

```diff
@@ -1,29 +1,62 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="Current" DefaultTargets="Build"
-  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Common Package Properties -->
   <PropertyGroup>
-    <MdsVersionDefault>7.0.0</MdsVersionDefault>
     <BuildNumber Condition="'$(BuildNumber)' == ''">0</BuildNumber>
+  </PropertyGroup>
+
+
+  <!-- Abstractions Package Properties -->
+  <PropertyGroup>
+    <!-- The current Major version number for the Abstractions package. -->
+    <AbstractionsPackageMajorVersion>1</AbstractionsPackageMajorVersion>
 
     <!--
-      GOTCHA: These two properties must appear in the first <PropertyGroup> in
-      this file due to hardcoded XML processing in:
+      The default Abstractions package version.
 
-      eng/pipelines/common/templates/jobs/validate-signed-package-job.yml
+      This may be overridden by specifying a value on the command-line:
+
```

Comment:
super nit: sparsely populated comments like these seems ... idk, um, unnecessarily un-dense. We could get rid of this extra line and it becomes just a bit more readable imo.